dash/frontend/src/app/modules/private/pages/reports/running-vulnerabilities/running-vulnerabilities.component.html

```diff
@@ -86,73 +88,75 @@
           <div class="page-table">
             <mat-table [dataSource]="dataSource">
               <ng-container matColumnDef="image" class="column-word-break">
-                <mat-header-cell *matHeaderCellDef>Image</mat-header-cell>
-                <mat-cell *matCellDef="let vulnerability">{{vulnerability.image}}</mat-cell>
+                <mat-header-cell class="min-width-8" *matHeaderCellDef>Image</mat-header-cell>
+                <mat-cell class="min-width-8" *matCellDef="let vulnerability">{{vulnerability.image}}</mat-cell>
```

Comment:
Does this work when the data in the rows is larger than the title, and larger than that min-width size?

Elsewhere we've been setting the min-width on the generated class name for the column, so I'm not sure if setting it directly on the cells will behave differently.

Reply:
It seemed to when I was testing it, the data breaks into a new line if necessary with the row heights resizing. I'm not sure if that's the behavior you expect on these tables.

If not I can look at defining it on generated class names.

Reply:
My concern was that the cells in the same columns might end up with different widths and mess up the alignment of the table. So if that's not the case it is good.

Reply:
Alright, yeah I haven't seen any instance where the columns have fallen out of alignment when testing locally. If something crops up during testing I can circle back to it, but so far it's looked good.